config/locales/en.yml

```diff
@@ -147,6 +147,7 @@ en:
       copyright_html: "&copy; Copyright 2010&ndash;%{to_date} Northwestern University"
       logo_alt: "Northwestern Logo"
       logo_file: "logo.jpg"
+      logo_link_to: "/"
```

Comment:
I think we should use an entry in `config/settings.yml` for this (something like `footer_logo_link: "/"`) and read it with `Settings.footer_logo_link` in the view.

The main reason for this is because it would otherwise require changes in all schools repos that override the footer locale content.